unittests/AST/ASTImporterTest.cpp

```diff
@@ -3352,5 +3451,9 @@ INSTANTIATE_TEST_CASE_P(
     ParameterizedTests, CanonicalRedeclChain,
     ::testing::Values(ArgVector()),);
 
+INSTANTIATE_TEST_CASE_P(
+    ParameterizedTests, ImportImplicitMethods,
+    ::testing::Values(ArgVector(), ArgVector{"-fdelayed-template-parsing"}),);
```

Comment:
If delayed-template-parsing needed (no templates in the test code)?

Reply:
I think it is better to leave it there, to guarantee that we avoid future errors if we add a template test to `ImportImplicitMethods`. Also, soon we will execute all tests with the same switch sets, see https://reviews.llvm.org/D47367